apps/src/templates/PaneHeader.jsx

```diff
@@ -82,7 +82,7 @@ var styles = {
  * A purple pane header that can have be focused (purple), unfocused (light purple)
  * or read only (charcoal).
  */
-var PaneHeader = React.createClass({
+const PaneHeader = Radium(React.createClass({
```

Comment:
They were already being wrapped before, except it was right before export. Now that I'm exporting on the same line as they are being declared, I had to move the Radium calls.